<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QcRuleSettingsDlg</class>
 <widget class="QWidget" name="QcRuleSettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1429</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QcRuleSettingsDlg</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_ruleSet">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>100</y>
     <width>983</width>
     <height>630</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QTableView" name="tableView_rules">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>983</width>
       <height>581</height>
      </rect>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>43</number>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="labelQcX">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>质控品X</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxQcX">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>80</y>
       <width>234</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelQcY">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>80</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>质控品Y</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxQcY">
     <property name="geometry">
      <rect>
       <x>645</x>
       <y>80</y>
       <width>234</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelTargetX">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>靶值</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSdX">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>标准差</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTargetY">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>150</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>靶值</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSdY">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>220</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>标准差</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTagetX">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>150</y>
       <width>234</width>
       <height>40</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSdX">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>220</y>
       <width>234</width>
       <height>40</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTargetY">
     <property name="geometry">
      <rect>
       <x>645</x>
       <y>150</y>
       <width>234</width>
       <height>40</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSdY">
     <property name="geometry">
      <rect>
       <x>645</x>
       <y>220</y>
       <width>234</width>
       <height>40</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>342</width>
     <height>630</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar::handle:vertical  {
    margin: 46px 3px 3px 2px;
}
QScrollBar::sub-line:vertical  {
    height: 43px; 
    border-bottom: 1px solid #a4a4a4;
}</string>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>43</number>
   </attribute>
  </widget>
  <widget class="QFrame" name="line_h_1">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>148</y>
     <width>983</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_2">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>729</y>
     <width>983</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>342</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>729</y>
     <width>342</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>1</width>
     <height>630</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_2">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>100</y>
     <width>1</width>
     <height>630</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_3">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>148</y>
     <width>1</width>
     <height>581</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_4">
   <property name="geometry">
    <rect>
     <x>1378</x>
     <y>148</y>
     <width>1</width>
     <height>581</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>757</y>
     <width>1330</width>
     <height>52</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1330</width>
     <height>52</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1330</width>
     <height>52</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>14</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1085</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSave">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Cancel">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
      <property name="buttons" stdset="0">
       <string>btn_dlg</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>frame</zorder>
  <zorder>tabWidget_ruleSet</zorder>
  <zorder>tableWidget</zorder>
  <zorder>line_h_1</zorder>
  <zorder>line_h_2</zorder>
  <zorder>line_h_3</zorder>
  <zorder>line_h_4</zorder>
  <zorder>line_v_1</zorder>
  <zorder>line_v_2</zorder>
  <zorder>line_v_3</zorder>
  <zorder>line_v_4</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
